### DISCUSSION TITLE
Dependent picklists

### DISCUSSION THREAD

**Discussion by willemmulder** [keystonejs/keystone#7815] [Feature requests]
This is coming from a discussion on Slack where we would like to have dependent picklists (or any sort of dependency between fields) where for example the picklist with States changes depending on the Country that one picks in another picklist.

For that to happen, fields should have access to the currently selected (but unsaved) values of other fields on the page. 

I'd like to think that it would be beneficial for a field to have access to the stored state of the whole record as well as the unsaved / current state of the whole record. So that it can change its values or behaviour based on that.

Would that be something worth considering?

**Comment by chukwumaijem** [comment]
Currently building an entire page to solve this one problem. This is a feature worth considering.
